layers/+misc/multiple-cursors/README.org

```diff
@@ -60,3 +59,43 @@ For easy navigation you also have the following:
 | ~C-t~       | evil-mc-skip-and-goto-next-match   |
 | ~C-M-j~     | evil-mc-make-cursor-move-next-line |
 | ~C-M-k~     | evil-mc-make-cursor-move-prev-line |
+|             |                                    |
```

Comment:
Replace the last empty table row with an empty line, to leave a space between the key binding table and the following header.

Reply:
Done